### PR TITLE
feat(box): add overflow-wrap prop

### PR DESCRIPTION
**Pull request by TheSisb**
https://github.com/twilio-labs/paste/discussions/1396

### CODE REVIEW

**Review by richbachman — APPROVED** [review]

File: packages/paste-core/primitives/box/src/index.tsx [hunk 1]
```diff
@@ -81,6 +90,15 @@ interface BoxBaseStyleProps
...
   clip?: ClipProperty;
+  backgroundAttachment?: BackgroundAttachmentProperty;
+  textAlign?: TextAlignProperty;
+  textTransform?: TextTransformProperty;
+  textDecoration?: TextDecorationProperty<string>;
+  textOverflow?: TextOverflowProperty;
+  whiteSpace?: WhiteSpaceProperty;
+  wordBreak?: WordBreakProperty;
+  wordWrap?: WordWrapProperty;
+  overflowWrap?: OverflowWrapProperty;
```
Comment (SiTaggart) [inline packages/paste-core/primitives/box/src/index.tsx:101]: Most of these are already declared in style props.

Overflow wrap should be added to layout style props

**Review by SiTaggart — CHANGES_REQUESTED** [review]

File: packages/paste-core/primitives/box/src/index.tsx (hunk 1 above)
Comment (TheSisb) [reply packages/paste-core/primitives/box/src/index.tsx]: Goood catch. Been a minute since I touched these files. Fixed.

Comment (richbachman) [reply packages/paste-core/primitives/box/src/index.tsx]: Totally forgot about those being there.

**Review by richbachman — APPROVED** [review]
Just need to add style-props to the changeset.

**Review by SiTaggart — APPROVED** [review]
yup, add style-props to the changesets and you're good

**Pull request merged**